README.rst

```diff
@@ -231,15 +241,17 @@ Configure timeout for the TCP connect when establishing a SSH connection.
 ``OPENWISP_CONNECTORS``
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-+--------------+--------------------------------------------------------------------+
-| **type**:    | ``tuple``                                                          |
-+--------------+--------------------------------------------------------------------+
-| **default**: | .. code-block:: python                                             |
-|              |                                                                    |
-|              |   (                                                                |
-|              |     ('openwisp_controller.connection.connectors.ssh.Ssh', 'SSH'),  |
-|              |   )                                                                |
-+--------------+--------------------------------------------------------------------+
++--------------+--------------------------------------------------------------------------------+
+| **type**:    | ``tuple``                                                                      |
++--------------+--------------------------------------------------------------------------------+
+| **default**: | .. code-block:: python                                                         |
+|              |                                                                                |
+|              |   (                                                                            |
+|              |     ('openwisp_controller.connection.connectors.ssh.Ssh', 'SSH'),              |
+|              |     ('openwisp_controller.connection.connectors.snmp.Snmp', 'SNMP'),           |
+|              |     ('openwisp_controller.connection.connectors.airos.snmp.Snmp', 'AiOS SNMP'),|
+|              |   )                                                                            |
```

Comment:
think we have to be explicit with the naming here, we need two different connectors, one for OpenWrt and one for AirOS. In the case of SSH we can let the update strategy code figure out which sub class to use.

This part will need to be changed (along other things) in future iterations but at least if we do it this way I believe it should allow us to start using this.